src/universal/mutations/UpdateReflectionContentMutation.js

```diff
@@ -38,41 +32,9 @@ const mutation = graphql`
   }
 `;
 
-export const updateReflectionContentUpdater = (payload: ?RecordSourceProxy, store: RecordSourceSelectorProxy) => {
```

Comment:
i wasn't a huge fan of this functor pattern, and in this case, it didn't actually do anything.

the optimistic response also didn't do anything.